runtime/compiler/CMakeLists.txt

```diff
@@ -67,6 +67,44 @@ add_custom_command(
 )
 add_custom_target(j9jit_tracegen DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/env/ut_j9jit.h)
 
+# JITSERVER_SUPPORT and protobuf
+if(JITSERVER_SUPPORT)
+	message(STATUS "JITServer is supported")
+	add_definitions(-DJITSERVER_SUPPORT)
+
+	include(FindProtobuf)
+	set(Protobuf_USE_STATIC_LIBS ON)
+	find_package(Protobuf REQUIRED)
+
+	if(Protobuf_FOUND)
+		include_directories(${Protobuf_INCLUDE_DIRS})
+		set(PROTOBUF_SHARED_LIBRARY_SUFFIX ${CMAKE_SHARED_LIBRARY_SUFFIX})
+		string(REPLACE "29" "" PROTOBUF_SHARED_LIBRARY_SUFFIX ${PROTOBUF_SHARED_LIBRARY_SUFFIX})
+		string(REPLACE "${PROTOBUF_SHARED_LIBRARY_SUFFIX}" "${CMAKE_STATIC_LIBRARY_SUFFIX}" Protobuf_LIBRARIES ${Protobuf_LIBRARIES})
```

Comment:
It is perhaps unfortunate that openj9 modifies `CMAKE_SHARED_LIBRARY_SUFFIX`, but the old value is available in `J9VM_OLD_SHARED_SUFFIX`; I suggest the replacement can and should be more specific:
```
	string(REPLACE
		"${CMAKE_SHARED_LIBRARY_PREFIX}protobuf${J9VM_OLD_SHARED_SUFFIX}"
		"${CMAKE_STATIC_LIBRARY_PREFIX}protobuf${CMAKE_STATIC_LIBRARY_SUFFIX}"
		Protobuf_LIBRARIES
		${Protobuf_LIBRARIES})
```
This avoids spreading knowledge of the `29` part of library names.